src/cmds/cloud/signin.js

```diff
@@ -38,16 +38,18 @@ const login = async (config) => {
     protocol: config.protocol,
   });
 
+  console.log(email, password);
+
   try {
-    const credentials = await client.authUser(email, password);
+    const credentials = await client.createToken(email, password);
     const credentialsLocation = `${os.homedir()}/.knot/credentials.json`;
     fse.outputFileSync(credentialsLocation, JSON.stringify(credentials, null, 2));
     console.log(colors.cyan(`You have been successfully logged in.\nCredentials saved in ${credentialsLocation}`));
   } catch (err) {
-    if (err.response.status === 401) {
+    if (err.code === 403) {
```

Comment:
This will be possible only by using a new version of `Authenticator`, which should includes this [change](https://github.com/CESARBR/knot-cloud-sdk-js-authenticator/commit/b91eb5b84b35522bd7d18bf93eccbc76ef6e2277).

Reply:
Done.